notebooks/tutorials/cloud-ml-engine/Model training and prediction with Cloud Machine Learning Engine.ipynb

```diff
@@ -0,0 +1,571 @@
+    "\n",
+    "Note: The model can take a few minutes to train and show up in Cloud Storage."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gsutil ls gs://$BUCKET_NAME/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Serve the model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the model is successfully created and trained, you can serve it. A model can have different versions. In order to serve the model, create a model and version in Cloud ML Engine.\n",
+    "\n",
+    "Define the model and version names:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MODEL_NAME = \"CensusPredictor\"\n",
+    "VERSION_NAME = \"census_predictor_{}\".format(int(time.time()))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create the model in Cloud ML Engine:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud ml-engine models create $MODEL_NAME --regions us-central1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create a version that points to your model file in Cloud Storage:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud ml-engine versions create $VERSION_NAME \\\n",
+    "  --model=$MODEL_NAME \\\n",
+    "  --framework=scikit-learn \\\n",
+    "  --origin=gs://$BUCKET_NAME/ \\\n",
+    "  --python-version=3.5 \\\n",
+    "  --runtime-version=1.9"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Make predictions\n",
+    "\n",
+    "### Format data for prediction\n",
+    "\n",
+    "Before you send an online prediction request, you must format your test data to prepare it for use by the Cloud ML Engine prediction service. Make sure that the format of your input instances matches what your model expects.\n",
+    "\n",
+    "Create an `input.json` file with each input instance on a separate line. The following example uses ten data instances. Note that the format of input instances needs to match what your model expects. In this example, the Census model requires 14 features, so your input must be a matrix of shape (`num_instances, 14`)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define a name for the input file\n",
+    "INPUT_FILE = \"./census_training/input.json\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%%writefile $INPUT_FILE\n",
```

Comment:
As noted in the previous comment, we need to replace this block with the first 10 instances from "adult.test".

Could we grab them from the cloud-ml-samples bucket, print them here, and then run the prediction request?